src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int MAXSIZE = 12;
+    private static final int DEFAULT_STACK_VALUE = -1;
+    private static final int MAX_STACK_VALUE = 11;

```

Comment:
Another constant. It is related to MAXSIZE (yet, the name is misinforming)
INDEX better than VALUE.